### PR TITLE
fix: add the other requested "Learn more" links

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This adds the remainder of the "Learn more" links as requested by the documentation team.

## Task Item

refs #7587

## Screenshots

image

### CODE REVIEW

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx [hunk 1]
```diff
@@ -243,8 +256,11 @@ export const editorSteps: { [key in ManifestEditorSteps]: EditorStep } = {
     editJson: false,
     content: AddCallers,
     subText: () =>
-      formatMessage(
-        'To ensure a secure connection, provide the App ID of the bots that can connect to your skill.  If you don’t have this information, you can also add this information in Skill Configuration. Learn more.'
+      formatMessage.rich(
+        'To ensure a secure connection, provide the App ID of the bots that can connect to your skill.  If you don’t have this information, you can also add this information in Skill Configuration. <link>Learn more.</link>',
+        {
+          link: makeLink('https://docs.microsoft.com/en-us/composer/how-to-connect-to-a-skill'),
```
Comment (tdurnford) [inline Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:262]: Does this need to be an aka link?

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx [hunk 2]
```diff
@@ -175,15 +177,26 @@ const validate = ({ content, schema }) => {
     }, {});
 };
 
+function makeLink(url: string) {
```
Comment (tdurnford) [inline Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:180]: nit: createLink sounds better to me... Doesn't really make a difference.

File: Composer/packages/client/src/pages/publish/Notifications.tsx [hunk 3]
```diff
@@ -117,11 +118,23 @@ export const getSkillPublishedNotificationCardProps = (
...
       item.status === 200
-        ? formatMessage(
-            'Keep this URL handy to share it with other developers to use in their bot projects. You can find this URL in the project settings tab.'
+        ? formatMessage.rich(
+            'Copy and share this URL with the bots that can connect to this skill. <link>Learn more.</link>',
+            {
+              link: ({ children }) => (
+                <Link
+                  key={'republish-skill'}
+                  href={'https://docs.microsoft.com/en-us/composer/how-to-export-a-skill#republish-the-skill-to-azure'}
```
Comment (tdurnford) [inline Composer/packages/client/src/pages/publish/Notifications.tsx:130]: Does this need to be an aka link?

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx [hunk 4]
```diff
@@ -175,15 +177,26 @@ const validate = ({ content, schema }) => {
...
+  return ({ children }) => (
+    <Link key={url} href={url} rel="noopener noreferrer" target="_blank">
```
Comment (tdurnford) [inline Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:182]: Would it be better to generate a unique key here? If we have the url add to multiple links later on, we might run into issues.

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx [hunk 5]
```diff
@@ -175,15 +177,26 @@ const validate = ({ content, schema }) => {
...
     content: Description,
     editJson: false,
     title: () => formatMessage('Export your bot'),
     subText: () =>
-      formatMessage(
-        'A skill is a bot that can perform a set of tasks one or more bots.  To make your bot available as a skill, it needs a manifest - a JSON file that describes the actions the skill can perform.'
+      formatMessage.rich(
+        'A skill is a bot that can perform a set of tasks one or more bots.  To make your bot available as a skill, it needs a manifest - a JSON file that describes the actions the skill can perform. <link>Learn more.</link>',
+        {
+          link: makeLink('https://docs.microsoft.com/en-us/composer/how-to-export-a-skill'),
```
[the remark this answers is not in the document]
Comment (beyackle) [reply Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:198]: I'm using the links that were handed to me; I think linking to `docs.microsoft.com` sites should be just as stable as the `aka.ms` ones.

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx (hunk 4 above)
Comment (beyackle) [reply Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:182]: The `key` prop is only here to make formatMessage stop throwing out warnings; they only need to be unique within a single formatMessage.rich call. If we ever need to put multiple links to the same URL in the same string, we can revisit this.

**Review by tdurnford — APPROVED** [review]

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx (hunk 2 above)
Comment (tdurnford) [inline Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:180]: Can you make this an arrow function?

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (srinaath) [reply Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:198]: @beyackle The links need to be aka.ms like all other links. This link might already have an aka.ms.
The reason is we get the flexibility to route aka.ms links to exact places we want. Like for ex if the export to skill has changed completely in Composer 3.0 we could just change what the aka.ms redirection points.

**Review by srinaath — CHANGES_REQUESTED** [review]

File: Composer/packages/client/src/pages/publish/Notifications.tsx (hunk 3 above)
Comment (srinaath) [reply Composer/packages/client/src/pages/publish/Notifications.tsx:130]: Same here. aka.ms link. Check this link as a sample
https://redirectiontool.trafficmanager.net/am/redirection/home?options=host:aka.ms&set=disableAkaUUHourly

File: Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (srinaath) [reply Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx:198]: https://aka.ms/bfcomposer-2-exportskill is the one i beleive that matches.

**Review by tdurnford — DISMISSED** [review]

**Review by tdurnford — DISMISSED** [review]

**Review by srinaath — APPROVED** [review]

**Review by tdurnford — APPROVED** [review]

**Pull request closed without merging**